Description: Account for rustls lacking default feature in Debian
 Due to absence of aws-lc-rs (and aws-lc).
 Needs another debcargo invocation to take effect.
Forwarded: not-needed
Author: Blair Noctis <ncts@debian.org>
Last-Update: 2025-03-31
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -108,2 +108,4 @@
 [dependencies.rustls]
+default-features = false
+features = ["ring", "logging", "std", "tls12"]
 version = "0.23"
